packages/server/tests/acceptance/rpc_batch2.spec.ts

```diff
@@ -40,741 +40,952 @@ import Address from '../../tests/helpers/constants';
+  });
+
+  describe('eth_estimateGas', async function () {
+    let basicContract;
 
     this.beforeAll(async () => {
-        requestId = Utils.generateRequestId();
-
-        accounts[0] = await servicesNode.createAliasAccount(40, relay.provider, requestId);
-        accounts[1] = await servicesNode.createAliasAccount(10, relay.provider, requestId);
-        accounts[2] = await servicesNode.createAliasAccount(10, relay.provider, requestId);
-        accounts[3] = await servicesNode.createAliasAccount(50, relay.provider, requestId);
-        contractId = await accounts[0].client.createParentContract(parentContractJson, requestId);
-
-        const params = new ContractFunctionParameters().addUint256(1);
-        contractExecuteTimestamp = (await accounts[0].client
-            .executeContractCall(contractId, 'createChild', params, 75000, requestId)).contractExecuteTimestamp;
-        tokenId = await servicesNode.createToken(1000, requestId);
-        logger.info('Associate and transfer tokens');
-        await accounts[0].client.associateToken(tokenId, requestId);
-        await accounts[1].client.associateToken(tokenId, requestId);
-        await servicesNode.transferToken(tokenId, accounts[0].accountId, 10, requestId);
-        await servicesNode.transferToken(tokenId, accounts[1].accountId, 10, requestId);
-
-        // allow mirror node a 5 full record stream write windows (5 sec) and a buffer to persist setup details
-        await new Promise(r => setTimeout(r, 5000));
-
-        // get contract details
-        mirrorContract = await mirrorNode.get(`/contracts/${contractId}`, requestId);
-
-        // get contract result details
-        mirrorContractDetails = await mirrorNode.get(`/contracts/${contractId}/results/${contractExecuteTimestamp}`, requestId);
-        mirrorSecondaryAccount = (await mirrorNode.get(`accounts?account.id=${accounts[1].accountId}`, requestId)).accounts[0];
+      basicContract = await servicesNode.deployContract(basicContractJson);
+    });
 
-        const latestBlock = (await mirrorNode.get(`/blocks?limit=1&order=desc`, requestId)).blocks[0];
-        blockNumberAtStartOfTests = latestBlock.number;
-        mirrorAccount0AtStartOfTests = await mirrorNode.get(`/accounts/${accounts[0].accountId}`, requestId);
+    it('@release should execute "eth_estimateGas"', async function () {
+      const res = await relay.call(RelayCalls.ETH_ENDPOINTS.ETH_ESTIMATE_GAS, [{}], requestId);
+      expect(res).to.contain('0x');
+      expect(res).to.not.be.equal('0x');
+      expect(res).to.not.be.equal('0x0');
     });
 
-    this.beforeEach(async () => {
-        requestId = Utils.generateRequestId();
+    it('@release should execute "eth_estimateGas" for contract call', async function () {
+      const expectedRes = `0x${(400000).toString(16)}`;
+      const estimatedGas = await relay.call(
+        RelayCalls.ETH_ENDPOINTS.ETH_ESTIMATE_GAS,
+        [
+          {
+            from: accounts[0].address,
```

Comment:
Good addition with `from` but why remove the `to` when the `data` field is for a function?
The test description doesn't seem to suggest that it's intentionally making a call with a missing to.

Reply:
Added the `to` back and the value returned is correct, however the following hardhat ethers call fails which maybe a separate issue:
```
 const estimatedGas = await ethers.provider.estimateGas({
            to: basicContract.address,
            data: BASIC_CONTRACT_PING_CALL_DATA,
            from: "0x67d8d32e9bf1a9968a5ff53b87d777aa8ebbee69"
        });
        
```